### ISSUE TITLE
Icons Render in Dark Grey when Dark Theme Enabled in FluentAvalonia NavigationView

### ISSUE THREAD

**Issue by nu11ptr** [davidxuang/FluentIcons#15]
Icons render in black/dark grey making them almost impossible to see vs. white/light gray like the builtin icons do

Avalonia: 11.1.4
FluentAvalonia: 2.1.0
FluentIcons.Avalonia.Fluent=1.1.263

**Comment by davidxuang** [comment]
![图片](https://github.com/user-attachments/assets/3dabba5b-f39f-4054-92c1-5ff1cda179b1)

Can't reproduce it. I didn't change the behaviour inherited from `FAIconElement` at all.

**Comment by nu11ptr** [comment]
My use case is with `NavigationView` and specifically with a menu item of type `NavigationViewItem`. I tried changing the `Foreground` property and it did nothing. I can provide a XAML code sample later if that helps.

**Comment by davidxuang** [comment]
![图片](https://github.com/user-attachments/assets/3b80b127-e518-419a-8271-95c5870f1492)

I still cannot reproduce even if I change the system theme on the fly.

**Comment by davidxuang** [comment]
![图片](https://github.com/user-attachments/assets/3b8dac77-4be1-4ab1-a39d-75b57f5843c9)

I don't think so. Try look elsewhere. I'm pretty sure I'm not messing with `Foreground` in this class and it inherits `FontIconSource` from upstream.

**Comment by nu11ptr** [comment]
I should probably also had my theme is on "Default", not "Dark" specifically if that makes a difference (so it turns "Dark" at night and "Light" during the day based on my system settings on my Mac).

This is my XAML - I can do a minimal reproduce later if this is too complicated:

```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:vm="using:ui.ViewModels"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:ui="clr-namespace:FluentAvalonia.UI.Controls;assembly=FluentAvalonia"
        xmlns:ic="clr-namespace:FluentIcons.Avalonia.Fluent;assembly=FluentIcons.Avalonia.Fluent"
        xmlns:system="clr-namespace:System;assembly=System.Runtime"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="ui.Views.MainWindow"
        x:DataType="vm:MainWindowViewModel"
        Icon="/Assets/avalonia-logo.ico"
        Title="Market Analyzer">

    <Design.DataContext>
        <vm:MainWindowViewModel />
    </Design.DataContext>

    <ui:NavigationView x:Name="NavView" IsPaneToggleButtonVisible="{Binding !$self.IsPaneOpen}"
                       MenuItemsSource="{Binding NavItems}">
        <ui:NavigationView.MenuItemTemplateSelector>
            <vm:NavItemTemplateSelector>
                <vm:NavItemTemplateSelector.HeaderTemplate>
                    <DataTemplate DataType="{x:Type system:String}">
                        <ui:NavigationViewItemHeader Content="{Binding .}" />
                    </DataTemplate>
                </vm:NavItemTemplateSelector.HeaderTemplate>

                <vm:NavItemTemplateSelector.ItemTemplate>
                    <DataTemplate DataType="{x:Type vm:NavItem}">
                        <ui:NavigationViewItem Content="{Binding Title}" Tag="{Binding Page}">
                            <ui:NavigationViewItem.IconSource>
                                <ic:SymbolIconSource Symbol="{Binding Icon}" />
                            </ui:NavigationViewItem.IconSource>
                        </ui:NavigationViewItem>
                    </DataTemplate>
                </vm:NavItemTemplateSelector.ItemTemplate>
            </vm:NavItemTemplateSelector>
        </ui:NavigationView.MenuItemTemplateSelector>
    </ui:NavigationView>
</Window>
```

**Comment by davidxuang** [comment]
That is a known issue with SkiaSharp working with `COLR` on MacOS. See #7.

TL;DR: Either enforce SkiaSharp 3, or use a workaround version of FluentIcons mentioned in that thread.

**Comment by nu11ptr** [comment]
Changing my version to `1.1.234-exp1` worked, thanks! I will do some research and see which workaround I want to use.
